cmd/root.go

```diff
@@ -75,6 +75,7 @@ func init() {
 	cloudctl cluster ls -o template --template "{{ .metadata.uid }}"
 
 	`)
+	rootCmd.PersistentFlags().BoolP("yes-i-really-mean-it", "", false, "skips security prompts (which can be dangerous to set blindly because actions can lead to data loss or additional costs)")
```

Comment:
please align with:
viper.GetBool("i-am-aware-of-dangerous-settings")

Reply:
Done.